deftree/svcparse/scanner.go

```diff
@@ -153,6 +153,8 @@ func BuildScanUnit(rr *RuneReader) (*ScanUnit, error) {
 						buf = append(buf, ch)
 						return setReturn(), nil
 					}
+				} else {
```

Comment:
Once again, maybe deserves a comment, as this fixes something but I have no idea why.